doc/api/stream.md

```diff
@@ -2,22 +2,44 @@
 
     Stability: 2 - Stable
 
-A stream is an abstract interface implemented by various objects in
-Node.js. For example a [request to an HTTP server][http-incoming-message] is a
-stream, as is [`process.stdout`][]. Streams are readable, writable, or both. All
-streams are instances of [`EventEmitter`][].
-
-You can load the Stream base classes by doing `require('stream')`.
-There are base classes provided for [Readable][] streams, [Writable][]
-streams, [Duplex][] streams, and [Transform][] streams.
-
-This document is split up into 3 sections:
-
-1. The first section explains the parts of the API that you need to be
-   aware of to use streams in your programs.
-2. The second section explains the parts of the API that you need to
-   use if you implement your own custom streams yourself. The API is designed to
-   make this easy for you to do.
+The `stream` module provides the abstract API for working with streaming data
```

Comment:
The original wording — "A stream is an abstract interface implemented by various objects in Node.js." — seems a bit more accurate here. The builtin module makes it easy to build objects that match the expected interface, and is used by Node internally to create stream objects — but it doesn't _provide_ the abstract API (folks can bring their own streams!)